finance_manager/src/main/java/com/manager/finance/user/service/ProfileServiceImpl.java

```diff
@@ -42,11 +38,31 @@ public Profile findByUserIdWithValidation() {
 
     private Profile createDefaultProfile() {
         User user = userService.getByUserHolder();
-        Sort sort = Sort.by("dtUpdate").descending();
-        Profile sourceProfile = profileDao.findProfileByUserId(user, sort);
-        Profile profile = profileMapper.sourceProfileToNewProfile(sourceProfile);
+        Profile profile = new Profile();
         profile.setIsDeleted(false);
         profile.setDtUpdate(TimeZoneUtils.getGmtCurrentDate());
-        return profileDao.save(profile);
+        profile.setUserId(user);
+        Profile createdProfile = profileDao.save(profile);
+        createDefaultCategories(createdProfile);
+        return createdProfile;
+    }
+
+    public void createDefaultCategories(Profile profile) {
+        categoryDao.save(new Category(
+                "Food", "#FF0000", TransactionTypeParameter.EXPENSE, profile));
+        categoryDao.save(new Category(
+                "Sport", "#FFA500", TransactionTypeParameter.EXPENSE, profile));
+        categoryDao.save(new Category(
+                "Pets", "#FFFF00", TransactionTypeParameter.EXPENSE, profile));
+        categoryDao.save(new Category(
+                "Utilities", "#3CB371", TransactionTypeParameter.EXPENSE, profile));
+        categoryDao.save(new Category(
+                "Сlothes", "#00FFFF", TransactionTypeParameter.EXPENSE, profile));
+        categoryDao.save(new Category(
+                "Salary", "#0000CD", TransactionTypeParameter.INCOME, profile));
+        categoryDao.save(new Category(
+                "Deposit interest", "#9932CC", TransactionTypeParameter.INCOME, profile));
+        categoryDao.save(new Category(
+                "Dividends by stock", "#C71585", TransactionTypeParameter.INCOME, profile));
     }
```

Comment:
Hope that it will be refactored soon, it's a incorrect way to create default categories